### PR TITLE
Update supported devices and VISA backends in the readme

### PR DESCRIPTION
**Pull request by adityakb**
## Proposed changes

Please include a summary of the changes and any links to related issues. Please also include relevant motivation and context.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Functionality update (non-breaking change which updates or changes existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] CI/CD update (an update to the CI/CD workflows, scripts, and/or configurations)
- [x] Documentation update (an update to enhance the user experience when reading through the docs)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have followed the guidelines in the [CONTRIBUTING](https://github.com/tektronix/tm_devices/blob/main/CONTRIBUTING.md) document
- [x] I have signed the CLA
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/tektronix/tm_devices/pulls) for the same update/change
- [x] I have performed a self-review of my code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Basic linting passes locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added necessary documentation (if appropriate)
- [x] I have updated the Changelog with a brief description of my changes

### CODE REVIEW

**Review by u625355 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ Things to be included in the next release go here.
 
 - Updated the `get_model_series()` function to only warn the user if the model is not found in the `SupportedModels` enumeration. This also eliminates false warnings during unit tests.
 
+### Added
+
+- Added TekVISA as one of the VISA backend support.
```
Comment (nfelt14) [inline CHANGELOG.md:27]:
```suggestion
- Added TekVISA as one of the VISA backends supported.
```

File: README.rst [hunk 2]
```diff
@@ -227,9 +229,10 @@ Supported Devices & Software Solutions
    :term:`SMUs <SMU>`, **24xx Standard**, :term:`PI`, ✅,
    , **24xx Interactive**, :term:`TSP`, ✅, ✅
    , **26xxB**, :term:`TSP`, ✅, 🚧
+   , **2636B**, :term:`TSP`, ✅, ✅
```
Comment (nfelt14) [inline README.rst:232]: Is this separate from the 26xxB models?

**Review by nfelt14 — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (adityakb) [reply README.rst:232]: Other SMU models under 26XXB have not been fully qualified. We would need additional investigation to determine the common APIs between these listed models. Given the full qualification on available SMU2636B we would want to make SMU2636B as fully qualified instrument.

File: README.rst [hunk 3]
```diff
@@ -118,6 +118,9 @@ dependencies on any operating system thanks to the
 `PyVISA-py <https://pyvisa.readthedocs.io/projects/pyvisa-py/en/latest/>`__
 VISA backend. However, in order to use any VISA functionality that
 PyVISA-py does not implement, a third-party VISA backend such as
+`TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__
```
Comment (nfelt14) [inline README.rst:121]: This section is not valid syntax for rST. Please verify that the docs build locally.

**Review by nfelt14 — CHANGES_REQUESTED** [review]

File: README.rst [hunk 4]
```diff
@@ -118,8 +118,11 @@ dependencies on any operating system thanks to the
...
+or
 `NI-VISA <https://www.ni.com/en-us/support/downloads/drivers/download.ni-visa.html>`__
-can be installed. ``tm_devices`` will use an available VISA backend if
+can be installed. v4.2.0 is the TekVISA version supported by ``tm_devices``.
```
Comment (nfelt14) [inline README.rst:124]: The minimum version for TekVISA needs to moved above to be next to the TekVISA link

**Review by nfelt14 — CHANGES_REQUESTED** [review]

File: README.rst [hunk 5]
```diff
@@ -118,8 +118,11 @@ dependencies on any operating system thanks to the
 `PyVISA-py <https://pyvisa.readthedocs.io/projects/pyvisa-py/en/latest/>`__
 VISA backend. However, in order to use any VISA functionality that
 PyVISA-py does not implement, a third-party VISA backend such as
+`TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__
```
Comment (nfelt14) [inline README.rst:121]:
```suggestion
`TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__ (>=4.2.0)
```

File: README.rst (hunk 4 above)
Comment (nfelt14) [reply README.rst:124]:
```suggestion
can be installed. ``tm_devices`` will use an available VISA backend if
```

File: README.rst [hunk 6]
```diff
@@ -118,8 +118,11 @@ dependencies on any operating system thanks to the
...
+or
 `NI-VISA <https://www.ni.com/en-us/support/downloads/drivers/download.ni-visa.html>`__
-can be installed. ``tm_devices`` will use an available VISA backend if
+can be installed. v4.2.0 is the TekVISA version supported by ``tm_devices``.
+``tm_devices`` will use an available VISA backend if
```
Comment (nfelt14) [inline README.rst:125]:
```suggestion
```

**Review by nfelt14 — CHANGES_REQUESTED** [review]

File: README.rst (hunk 3 above)
Comment (adityakb) [reply README.rst]: Run and Verified
image

File: README.rst (hunk 4 above)
Comment (adityakb) [reply README.rst]: Done.

**Review by u625355 — DISMISSED** [review]

**Review by nfelt14 — DISMISSED** [review]

**Review by nfelt14 — APPROVED** [review]

**Review by lekevin678 — APPROVED** [review]

**Review by michaelwagoner — APPROVED** [review]

**Pull request merged**